js/script.js

```diff
@@ -8,8 +8,10 @@ const BUTTON_UP_OUTLINE = "";
 const BUTTON_DOWN_COLOR = "#f0f0f0";
 const BUTTON_DOWN_OUTLINE = "1px solid blue";
 const CANVAS_INIT_COLOR = palette_color_array[0];
+let ACTIVE_COLOR_SELECT = "1nd-Color";
 const STATE = {
-    "activeColor": palette_color_array[palette_color_array.length - 1],
+    "1nd-Color": palette_color_array[palette_color_array.length - 1],
```

Comment:
Small nitpick, but can you rename `1nd-Color` and `2nd-Color` to `firstColor` and `secondColor` respectively? It's best to keep the camelCase consistent in at least this STATE object.